Automattic/wp-calypso
# Store Services: Add USPS shipping label signature requirement

**NOTE: Do not merge until `add/usps-signature-confirmation` lands!**

---

Test with WCS server branch `add/usps-signature-confirmation`.

Visual changes:

![](https://cldup.com/-x2CvCcPir-3000x3000.png)

To test:
* Verify that selecting "yes" results in a $2.55 increase in rate
* Verify that selecting "yes, from an adult" results in a $6.10 increase in rate
* Verify that label purchase succeeds
